bubblewrap.c

```diff
@@ -3148,7 +3148,7 @@ main (int    argc,
       if (intermediate_pids_sockets[PIPE_READ_END] != -1)
```

Comment:
nit: commit message should be `cleanup_fdp` instead of `cleanup_fd`?

Reply:
Indeed